pkg/drivers/commonconfig.go

```diff
@@ -328,6 +328,8 @@ func GetUpgradeInfo(ctx context.Context, operatorConfig utils.OperatorConfig, dr
 // GetConfigMap get configmap
 func GetConfigMap(ctx context.Context, cr csmv1.ContainerStorageModule, operatorConfig utils.OperatorConfig, driverName csmv1.DriverType) (*corev1.ConfigMap, error) {
 	log := logger.GetLogger(ctx)
+	var podmanLogFormat string
```

Comment:
Are there tests in commonfig_test.go that can be updated or added to reflect these changes?

Reply:
there are tests for getting the configmaps
nothing to be added more in this section